can/interfaces/serial/serial_can.py

```diff
@@ -12,35 +17,92 @@
 try:
     import serial
 except ImportError:
-    logger.error("You won't be able to use the serial can backend without the serial module installed!")
+    logger.error("You won't be able to use the serial can backend without the "
```

Comment:
For consistency with the other interfaces can you please switch this to use the `warning` level here.